src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -455,3 +466,14 @@ COORD Terminal::_ConvertToBufferCell(const COORD viewportPos) const
     THROW_IF_FAILED(ShortAdd(positionWithOffsets.Y, gsl::narrow<SHORT>(_ViewStartIndex()), &positionWithOffsets.Y));
     return positionWithOffsets;
 }
+
+// Method Description:
+// - Given two points in the buffer space, select the text only because we do not want to modify to the buffer in Terminal
+// Arguments:
+// - coordSelectionStart - Anchor point (start of selection) for the region to be colored
+// - coordSelectionEnd - Other point referencing the rectangle inscribing the selection area
+// - attr - Not used.
+void Terminal::ColorSelection(const COORD coordSelectionStart, const COORD coordSelectionEnd, const TextAttribute)
+{
+    SelectNewRegion(coordSelectionStart, coordSelectionEnd);

```

Comment:
Maybe make this just throw since it doesn't do what's expected and probably shouldn't/can't actually get called yet? THROW_HR(E_NOTIMPL)?

Reply:
changed